retro/sadtalker.py

```diff
@@ -39,6 +41,44 @@
 MAX_RES = 1920 * 1080
 
 
+# the original sadtalker function does not work for short audio
+def fixed_generate_blink_seq_randomly(num_frames):
+    ratio = np.zeros((num_frames, 1))
+    if int(num_frames / 2) <= 11:
+        return ratio
+    frame_id = 0
```

Comment:
Can you call the old fn here instead of rewrite (don't wanna own this code)

Reply:
Hmm ok. I’ll submit it as a pr upstream to the sadtalker repo instead

Reply:
no i meant since you only added a condition at the top, you can call the original function from the lib before overwriting it

Reply:
[bce1ef9](https://github.com/GooeyAI/gooey-gpu/pull/26/commits/bce1ef96bdd3b3de22fc70a6a6038a2c04eaf6c8)